### ISSUE TITLE
A11y_code.visualstudio.com_Docs_LeftRightNavigation_ScreenReader: Unclear functionality of the expand/collapse links for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2810]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 1.3.1, MAS 4.2.12

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/docs](https://code.visualstudio.com/docs)
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Docs" link.
3.	Left and right navigation will appear.
4.	Start the screen reader.
5.	Navigate to links such as “Setup”, “Get Started” and activate it.
6.	Listen if its state is announced or not.

**Actual:**
When screen reader users navigate to the links such as “Setup”, “Get Started” and activate it some content gets expanded or collapsed but the same is not notified to the screen reader users.

This issue is observed with "Edge + Narrator" and "Chrome + NVDA".

**Expected:**
When screen reader users activate the links such as “Setup”, “Get Started”, the expand/collapse state should be announced.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
)
**User Impact:**
The screen reader users finds it difficult to understand the state if the content has "collapsed/expanded" on activating the interactive elements.

**MAS Reference:**
MAS 1.3.1 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

MAS 4.2.12 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b19c05bd-d82f-4d6d-a8e5-7081bf5c2dba}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b19c05bd-d82f-4d6d-a8e5-7081bf5c2dba})
 
**Attachment for Reference:**
[Code.visualstudio.com_Docs_Left RightNavigation_ScreenReader Unclear expandcollapse links.pptx](https://github.com/microsoft/vscode-docs/files/3325135/Code.visualstudio.com_Docs_Left.RightNavigation_ScreenReader.Unclear.expandcollapse.links.pptx)

**Comment by octref** [comment]
Think this is fixed. Screen reader does announce expanded / collapsed states for the links, and the links are read as "Node.js / JavaScript drop down". The "drop down" parts indicates this is a expandable navigation.

![Screen Shot 2019-07-09 at 11 37 32 AM](https://user-images.githubusercontent.com/4033249/60914227-f514da00-a23d-11e9-9acb-9df7ffdbb4d5.png)

![image](https://user-images.githubusercontent.com/4033249/60914318-268da580-a23e-11e9-8453-8f43bddc9aa6.png)
